MapboxVisionAR.podspec

```diff
@@ -1,7 +1,7 @@
 Pod::Spec.new do |s|
 
   s.name         = "MapboxVisionAR"
-  s.version      = "0.3.0"
+  s.version      = "0.3.1"
```

Comment:
Same here. Version bump is a separate process.

Reply:
Okay. I will revert this.